crates/orderbook/openapi.yml

```diff
@@ -51,6 +51,32 @@ paths:
           application/json:
             schema:
               $ref: "#/components/schemas/OrderCreation"
+    delete:
+      summary: Cancels multiple orders by marking it invalid with a timestamp.
+      description: |
+        The successful deletion might not prevent solvers from settling the orders
+        Authentication must be provided by providing an EIP-712 signature of an
+        "OrderCacellations(bytes[] orderUids)" message.
```

Comment:
> The successful deletion might not prevent solvers from settling the orders

This is not very confidence inspiring for the user. 😅 
Do you think describing the circumstances where that can happen a bit more would give a user a better understanding of what to expect?

Also the use of punctuation seems inconsistent in this PR and probably in the openapi file as a whole.

Reply:
Oops, missing a `.` for sure.

Reply:
Fixed punctuation and reworded a bit.